### PR TITLE
Update forum database app for RDB refresh

### PR DESCRIPTION
**Pull request by karlud**

### CODE REVIEW

File: vagrant/forum/forum.py [hunk 1]
```diff
@@ -40,68 +38,26 @@
 
 # HTML template for an individual comment
 POST = '''\
-    <div class=post><em class=date>%(time)s</em><br>%(content)s</div>
+    <div class=post><em class=date>%s</em><br>%s</div>
```
Comment (forbiddenvoid) [inline vagrant/forum/forum.py]: Are we sticking with %s for string interpolation over using `.format()`?   f-strings are now part of Python 3.6 (https://www.python.org/dev/peps/pep-0498/), so I wonder if using the `.format()` will help future proof the code a little. 

I think the %s interpolation is in the course content, though, so there's an argument for keeping it as is for now.

File: vagrant/forum/forumdb.py [hunk 2]
```diff
@@ -1,31 +1,15 @@
...
-# Database access functions for the web forum.
-# 
+# "Database code" for the DB Forum.
 
-import time
+import datetime
 
-## Database connection
-DB = []
+POSTS = [("This is the first post.", datetime.datetime.now())]
```
Comment (forbiddenvoid) [inline vagrant/forum/forumdb.py]: I'm curious about your thought process here in using tuples over dicts. Is this just to reduce complexity?

File: vagrant/forum/solution/forumdb_solved.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+# Database code for the DB Forum, full solution!
+
+import psycopg2, bleach
+
+DBNAME = "forum"
+
+def get_posts():
```
Comment (forbiddenvoid) [inline vagrant/forum/solution/forumdb_solved.py]: What are your thoughts about adding a refactor step after this one? Both of these functions have some common code that probably could be factored out (the connection, cursor, execution, and close.

File: vagrant/forum/forumdb.py (hunk 2 above)
Comment (karlud) [inline vagrant/forum/forumdb.py]: Yep. Also it's how it is in the video.

File: vagrant/forum/forum.py (hunk 1 above)
Comment (karlud) [inline vagrant/forum/forum.py]: There were %'s in a bunch of the original videos that didn't get redone. The same warnings apply to using {} formats though.

File: vagrant/forum/solution/forumdb_solved.py (hunk 3 above)
Comment (karlud) [inline vagrant/forum/solution/forumdb_solved.py]: Sounds like a good idea for an extra text node after the forum quizzes?

File: vagrant/forum/forum.py (hunk 1 above)
Comment (karlud) [inline vagrant/forum/forum.py]: Added to the lesson:

By the way, the warnings about using `%` with SQL statements and user data also apply to newer ways of doing string formatting in Python. This includes [f-strings, introduced in Python 3.6](https://docs.python.org/3/reference/lexical_analysis.html#f-strings).

**Pull request merged**